model/task/db.go

```diff
@@ -2692,3 +2692,43 @@ func enableDisabledTasks(taskIDs []string) error {
 		})
 	return err
 }
+
+type NumExecutionsForIntervalInput struct {
+	ProjectId    string
+	BuildVarName string
+	TaskName     string
+	Requesters   []string
+	StartTime    time.Time
+	EndTime      time.Time
+}
+
+func CountNumExecutionsForInterval(input NumExecutionsForIntervalInput) (int, error) {
+	query := bson.M{
+		ProjectKey:      input.ProjectId,
+		BuildVariantKey: input.BuildVarName,
+		DisplayNameKey:  input.TaskName,
+		StatusKey:       bson.M{"$in": evergreen.TaskCompletedStatuses},
```

Comment:
Why do we want to filter this on TaskCompletedStatuses? If a task is in it's third execution, and the third one is still running, don't we still want to see 3 as the number of executions?

Reply:
following 
what do we expect to happen if the task is aborted or unscheduled?

Reply:
That makes sense. I would advocate to use a route name that makes it clear that it's the number of `completed` executions so that someone doesn't accidentally use it if it doesn't match their use case. 

Maybe something like `num_completed_executions` or `completed_executions_count`. It's definitely more clunky but I think it's worth it.

Reply:
I'll do this for the response variable name instead maybe -- that way the route name will still work if we do want to start allowing for status to be modifiable or etc, and also will make the variable itself clearer if we ever add more